### PR TITLE
Added Thunderbird version 91.0.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24575)

### CODE REVIEW

File: manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Mozilla.Thunderbird
+PackageVersion: "91.0"
+PackageName: Thunderbird
```
Comment (ItzLevvie) [inline manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml:4]:
```suggestion
PackageName: Mozilla Thunderbird
```

File: manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Publisher: Mozilla
+Moniker: thunderbird
+Tags:
+- thunderbird
+- email
+- eMail
+- E-mail
+- email Client
```
Comment (ItzLevvie) [inline manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml:12]:
```suggestion
- thunderbird
- email
- e-mail
- email-client
```

File: manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ShortDescription: Thunderbird is a free email application thatâ€™s easy to set up and customize!
```
Comment (ItzLevvie) [inline manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml:13]:
```suggestion
ShortDescription: Thunderbird is a free email application that's easy to set up and customize!
```

File: manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+PackageUrl: https://www.thunderbird.net
+License: Mozilla Public License Version 2.0
+LicenseUrl: https://www.mozilla.org/en-US/MPL/2.0/
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download-installer.cdn.mozilla.net/pub/thunderbird/releases/91.0/win64/en-US/Thunderbird%20Setup%2091.0.msi
+  InstallerSha256: 8CDEB0747642803F92A25CFFEF3CA01E193496CBAF2D12974370C315D282009C
```
Comment (ItzLevvie) [inline manifests/m/Mozilla/Thunderbird/91.0/Mozilla.Thunderbird.yaml:20]:
```suggestion
  InstallerSha256: 8CDEB0747642803F92A25CFFEF3CA01E193496CBAF2D12974370C315D282009C
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/24575#issuecomment-897740573

**Pull request closed without merging**